packages/legend-application-studio/src/stores/graphModifier/DomainGraphModifierHelper.ts

```diff
@@ -90,6 +91,12 @@ export const class_addProperty = action(
   },
 );
 
+export const class_arrangeProperty = action(
```

Comment:
I think these methods should now be renamed to `*_swap*` and using a plural, e.g. `class_swapProperties`